src/FritzBox/Api.php

```diff
@@ -57,7 +57,7 @@ public function getSID()
     private function getClient(): Client
     {
         if (!$this->client) {
-            $this->client = new Client($this->getClientOptions());
+            $this->client = new Client($this->getClientOptions(['verify' => false]));
```

Comment:
Repeating myself here. I won't merge this as its inherently insecure. Feel free to make this a configurable option that is being read inside `getClientOptions`.

Reply:
Well, as a standard php installation usually never trust the self-signed certificates of FritzBoxes it is most likely the common case that you need to add this option to use https for accessing the FB.
Of course I can add another configurable option (that I would enable by default) so that the FB's SSL certificate won't get checked, I just don't see the practical point...

Reply:
It is insecure. This PR wants to use ftps and makes it default, yet wants to accept untrusted certficates. Thats contradictory. I'm accessing my FB over the internet and it has a perfectly validatable certificate. If this becomes the default I'm opening my installation to misuse.

Reply:
Well, that's a point. If you are using this script over the Internet you are right.
I'll add this config option soon.